src/components/RequestGroupComposition/Request.vue

```diff
@@ -165,10 +173,25 @@
               label-size="sm"
               label-align-sm="right"
               label-cols-sm="4"
-              label=""
+              :label="getFromObject(formConfig, ['request', 'mosaic', 'label'], '')"
               label-for="mosaic-button"
+              :invalid-feedback="
+                getFromObject(
+                  formConfig,
+                  ['request', 'mosaic', 'invalid_parameters_feedback'],
+                  `The limit to the number of mosaic pointings that can be generated is ${mosaicMaxNumPointings}. Please
+                  update your parameters before generating a mosaic.`
```

Comment:
Should we mention mosaics with "unlimited" pointings can be submitted through the API, or is it not really worth it?

Reply:
I think I left this description more general in case someone overrides their API to change how many pointings are allowed which would make this message incorrect, and in the case that they don't notice this message needs to be updated, it would just be wrong until they did notice which could be annoying. I guess though if they do override their API they'll likely want to update this message anyway, but maybe not... Given that, I'll change this message to say that this limit is in the frontend, so at least it's a bit more clear where this limit is applied.